set.mm

```diff
@@ -419171,116 +419164,181 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
-       blank lines.  If there is a "@( Begin $[ ... $] @)" comment (where "@"
-       is actually "$") before the section header, then the double blank line
-       should go before that comment.
+       The file set.mm should have a double blank line between sections, and at
+       no other places.  In particular, there are no triple blank lines.
        </li>
        <li>
        The header comments should be spaced as those of Part 1, namely, with
        a blank line before and after the comment, and an indentation of two
        spaces.
        </li>
        <li>
-       Header comments are not rewrapped by the Metamath program [as of
-       24-Oct-2021], but similar spacing and wrapping should be used as for
-       other comments: double spaces after a period ending a sentence, line
-       wrapping with line width of 79, and no trailing spaces at the end of
-       lines.
+       As of 20-Sep-2022, section comments are not rewrapped by the Metamath
+       program "MM> WRITE SOURCE set.mm / REWRAP" command, though this is
+       expected in a future version.  Similar spacing and wrapping should be
+       used as for other comments: double spaces after a period ending a
+       sentence, line wrapping with line width of 79, and no trailing spaces
+       at the end of lines.
+       </p>
        </li>
        </ul>
+
+       <li><b>Contributors.</b>
+       <p>
+       Each assertion (theorem, definition or axiom) has a contribution tag of
+       the form "(Contributed by xxx, 00-Xxx-0000)" (see Metamath Book,
+       p. 142).  The date cannot serve as a proof of anteriority since there is
+       currently no formal guarantee that the date is correct (a claim of
+       anterioty can be backed, for instance, by the uploading of a result to a
+       public repository with verifiable date).  The contributor is the first
+       person who proved (or stated, in the case of a definition or axiom) the
+       statement.  The list of contributors appears at the beginning of set.mm.
+       </p>
+       <p>
+       An exception should be made if a theorem is essentially an extract or a
+       variant of an already existing theorem, in which case the contributor
+       should be that of the statement from which it is derived, with the
+       modification signaled by a "(Revised by xxx, 00-Xxx-000)" tag.  Since
+       this is somewhat subjective, judgment and intellectual honesty should be
+       applied, with collegial settlement in case of dispute.
+       </p>
        </li>
 
-       <li><b>Format of section headers.</b>
-       The database set.mm has a sectioning system with four levels of titles,
-       indicated by "decoration lines" which are 79-character long repetitions
-       of ####, #*#*, =-=-, and -.-. (in descending order of sectioning level).
+       <li><b>Usage of parentheticals.</b>
+       <p>
+       Usually, the comment of a theorem should contain at most one of the
```

Comment:
A revision tag should have a reason added to it, like "Avoid ax-10."  That is also a good reason to not collapse several revisions from the same person into a single one.

Reply:
Indeed. Added. There are obviously a few conventions yet to be documented.